test/integration/rake_task_logs_request_with_active_record_test.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class RakeTaskLogsRequestWithActiveRecordTest < ActionDispatch::IntegrationTest
+  setup do
+    RakeUi.configuration.active_storage = true
+
+    @file = Rack::Test::UploadedFile.new(Rails.root.join('fixtures/sample_log.txt'), 'text/plain')
+    @rake_task_log = ::RakeTaskLog.create!(
+      status: 1,
+      name: 'Test Rake Task',
+      args: 'arg1,arg2',
+      environment: 'test',
+      rake_command: 'rake test',
+      rake_definition_file: 'definition_file.rake',
+      log_file: @file,  # Attach the file
+      log_file_name: 'sample_log.txt',
+      log_file_full_path: '/path/to/sample_log.txt',
+      raker_id: 'rake-id'
+    )
```

Comment:
Why use an @ var instead of `let`?  

```suggestion
  let(:file) { Rack::Test::UploadedFile.new(Rails.root.join('fixtures/sample_log.txt'), 'text/plain') }
  let(:rake_task_log) do
    ::RakeTaskLog.create!(
      status: 1,
      name: 'Test Rake Task',
      args: 'arg1,arg2',
      environment: 'test',
      rake_command: 'rake test',
      rake_definition_file: 'definition_file.rake',
      log_file: @file,  # Attach the file
      log_file_name: 'sample_log.txt',
      log_file_full_path: '/path/to/sample_log.txt',
      raker_id: 'rake-id'
    )
  end

  setup do
    RakeUi.configuration.active_storage = true
    rake_task_log
```

Reply:
please, thank you for your reply to this before merging/accepting the suggestion.

Reply:
@NewAlexandria actually it's showing me error `undefined method let for RakeTaskLogsRequestWithActiveRecordTest:Class`
since we're using integration test in `minitest` we cannot use **let** here, it's specific to rspec
however I removed **@** since instance var is not needed here

Reply:
I'm not sure what you mean; we have `let` in most or all of the tests in the SC main repo.